src/edit/DistortableImage.EditToolbar.js

```diff
@@ -1,182 +1,224 @@
+        tooltip;
+
+      if (edit._mode === 'lock') {
+        icon = 'lock_open';
+        tooltip = 'Unlock';
+      } else {
+        icon = 'lock';
+        tooltip = 'Lock';
+      }
+
+      options = options || {};
+      options.toolbarIcon = {
+        html: '<i class="material-icons-outlined md-18">' + icon + '</i>',
+        tooltip: tooltip
+      };
+
+      EditOverlayAction.prototype.initialize.call(this, map, overlay, options);
+    },
+
+    addHooks: function() {
+      var editing = this._overlay.editing;
+
+      editing._toggleLock();
+      this.disable();
+    }
 	}),
-
-	ToggleEditable = EditOverlayAction.extend({
-		options: { toolbarIcon: {
-			html: '<span class="fa fa-lock"></span>',
-			tooltip: 'Lock / Unlock editing',
-			title: 'Lock / Unlock editing'
-		}},
-
-		addHooks: function() {
-			var editing = this._overlay.editing;
-
-			editing._toggleLock();
-			this.disable();
-		}
+	
+  ToggleRotateScale = EditOverlayAction.extend({
+    initialize: function(map, overlay, options) {
+			var edit = overlay.editing,
+				icon,
+				tooltip; 
+
+        if (edit._mode === 'rotateScale') {
+					icon = 'transform';
+					tooltip = 'Distort';
+				} else {
+					icon = 'crop_rotate';
+					tooltip = 'Rotate+Scale';
+				}
+
+      options = options || {};
+      options.toolbarIcon = {
+        html: '<i class="material-icons md-18">' + icon + '</i>',
+        tooltip: tooltip
+      };
+
+      EditOverlayAction.prototype.initialize.call(this, map, overlay, options);
+    },
+
+    addHooks: function() {
+      var editing = this._overlay.editing;
+
+      editing._toggleRotateScale();
+      this.disable();
+    }
 	}),
+	
+  ToggleExport = EditOverlayAction.extend({
+    options: {
+      toolbarIcon: {
+        html: '<i class="material-icons md-18">get_app</i>',
+        tooltip: 'Export Image'
+      }
+    },
+
+    addHooks: function() {
+      var editing = this._overlay.editing;
+
+      editing._toggleExport();
+      this.disable();
+    }
+	}),
+	
+  ToggleOrder = EditOverlayAction.extend({
+    initialize: function(map, overlay, options) {
+      var edit = overlay.editing,
+        icon,
+        tooltip;
+
+        if (edit._toggledImage) {
+          icon = 'flip_to_front';
+          tooltip = 'Stack to front';
+        } else {
+          icon = 'flip_to_back';
+          tooltip = 'Stack to back';
+        }
+
+      options = options || {};
+      options.toolbarIcon = {
+        html: '<i class="material-icons md-18">' + icon + '</i>',
+        tooltip: tooltip
+      };
+
+      EditOverlayAction.prototype.initialize.call(this, map, overlay, options);
+    },
+
+    addHooks: function() {
+      var editing = this._overlay.editing;
+
+      editing._toggleOrder();
+      this.disable();
+    }
+	}),
+	
+  EnableEXIF = EditOverlayAction.extend({
+    options: {
+      toolbarIcon: {
+        html: '<i class="material-icons-outlined md-18">explore</i>',
+        tooltip: 'Geocode Image'
```

Comment:
Just a minor doubt, should this be "geolocate image" instead, or do you think it's good as is?

Reply:
done!